src/System.Runtime.InteropServices.RuntimeInformation/src/ArchitectureManager.Unix.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.ComponentModel;
+
+namespace System.Runtime.InteropServices
+{
+    internal partial struct ArchitectureManager
```

Comment:
Do we really need an "architecture manager"? This should be simple enough to implement directly in RuntimeInformation.